app/models/ReviewByGoldenTicket.scala

```diff
@@ -475,4 +476,24 @@ object ReviewByGoldenTicket {
       }
       tx2.exec()
   }
+
+  def markProposalReviewAsDelayed(uuid: String, proposalId: String, reason: String): Unit = Redis.pool.withClient {
+    client =>
+      client.hset(s"ReviewGT:DelayedReviews:ByEventCode:${ConferenceDescriptor.current().eventCode}:AndAuthor:${uuid}", proposalId, reason)
+  }
+
+  def proposalDelayedReviewReason(uuid: String, proposalId: String): Option[String] = Redis.pool.withClient {
+    client =>
+      client.hget(s"ReviewGT:DelayedReviews:ByEventCode:${ConferenceDescriptor.current().eventCode}:AndAuthor:${uuid}", proposalId)
+  }
+
+  def delayedReviewsReasons(uuid: String) = Redis.pool.withClient {
```

Comment:
Please add the return type for all public function

Reply:
fixed through f7adcba1